src/game/RandomizationOfShipsPositions.js

```diff
@@ -86,40 +88,53 @@ const positionShipHorizontally = (shipIndex, direction) => {
   if (
     gameBoardArray[position[0]]
       .slice(position[1], position[1] + SHIPS_LIST[shipIndex].size)
-      .some((el) => el.isEmpty === false)
+      .some((el) => {
```

Comment:
One line return doesn't need `{ ... }` or even `return` statement. Of course, adding them isn't a mistake. But if an element is an object then doest `el.isEmpty === false` gives you anything to that test 😄 ? And one other thing `el.isEmpty === false` is equal to `!el.isEmpty`

Reply:
Yeah, my fail with this resturn
It gives me beacuse isEmpty can be truthy so we will need rerun function. I can get object with isEmpty = false so i can far or receive object with isEmpty = true and i need run again this function to get new position for ship
Yeah, it's equal... 🤦

Reply:
fixed